### ISSUE TITLE
Difference from NAE

### ISSUE THREAD

**Issue by zsl1997** [DeanChan/HOIM-PyTorch#6]
I would like to ask what is the difference between this code and your other NAE?

**Comment by dichen-cd** [comment]
The two repositories share most of the backbone codes.
The main differences are the loss function and post-process.
This repository implements our HOIM loss, while the NAE repo implements the Norm-Aware Embedding method.
